api/v1alpha1/ext_auth_types.go

```diff
@@ -33,6 +33,8 @@ type ExtAuth struct {
 	// +optional
 	HeadersToExtAuth []string `json:"headersToExtAuth,omitempty"`
 
+	BodyToExtAuth *BodyToExtAuth `json:"bodyToExtAuth,omitempty"`
```

Comment:
Nit: can you add a line of comment to explain this field?
Also mark it as `+optional`